aspnetcore/fundamentals/error-handling/samples/8.x/ErrorHandlingSample/CustomExceptionHandler.cs

```diff
@@ -1,19 +1,23 @@
-﻿using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Diagnostics;
 
 namespace ErrorHandlingSample
 {
     public class CustomExceptionHandler : IExceptionHandler
     {
-        private ILogger<CustomExceptionHandler> logger;
-        public CustomExceptionHandler(ILogger<CustomExceptionHandler> logger){
+        private readonly ILogger<CustomExceptionHandler> logger;
+        public CustomExceptionHandler(ILogger<CustomExceptionHandler> logger)
+        {
             this.logger = logger;
         }
-        public ValueTask<bool> TryHandleAsync(HttpContext httpContext, Exception exception, CancellationToken cancellationToken)
+        public ValueTask<bool> TryHandleAsync(
+            HttpContext httpContext,
+            Exception exception,
+            CancellationToken cancellationToken)
         {
             var exceptionMessage = exception.Message;
             logger.LogError("Error Message: {exceptionMessage}", exceptionMessage);
             logger.LogError("Time of occurrence {time}", DateTime.Now);
-            // returned false to continue with the default behavior
+            // Return false to continue with the default behavior
```

Comment:
```suggestion
            // Return false to continue with the default behavior
            // - or - return true to signal that this exception is handled
```
Repeat that valueable piece of information in this comment?